packages/terra-clinical-result/src/common/observation/_Observation.jsx

```diff
@@ -48,17 +43,24 @@ const defaultProps = {
   hideUnit: false,
 };
 
-const interpretationIndicators = {
-  CRITICAL: <IconCritical className={cx('icon-interpretation')} />,
-  EXTREMEHIGH: <IconCritical className={cx('icon-interpretation')} />,
-  EXTREMELOW: <IconCritical className={cx('icon-interpretation')} />,
-  PANICHIGH: <IconCritical className={cx('icon-interpretation')} />,
-  PANICLOW: <IconCritical className={cx('icon-interpretation')} />,
-  VABNORMAL: <IconCritical className={cx('icon-interpretation')} />,
-  POSITIVE: <IconCritical className={cx('icon-interpretation')} />,
-  ABNORMAL: <IconAbnormal className={cx('icon-interpretation')} />,
-  HIGH: <IconHigh className={cx('icon-interpretation')} />,
-  LOW: <IconLow className={cx('icon-interpretation')} />,
+const interpretationIndicatorMap = {
+  critical: <IconCritical className={cx('icon-interpretation')} />,
+  'critical-high': <IconCritical className={cx('icon-interpretation')} />,
+  'critical-low': <IconCritical className={cx('icon-interpretation')} />,
+  positive: <IconCritical className={cx('icon-interpretation')} />,
```

Comment:
Should positive be using `IconCritical`?

Reply:
yes - "Positive" shares the same visual style as Critical.